xdsl/parser.py

```diff
@@ -123,6 +123,10 @@ def __post_init__(self):
         else:
             self._pos = Position(self.str)
 
+    def get_pos(self) -> Position | None:
```

Comment:
How is this change related to this PR? It does not seem to get used, in contrast, it is not used where it could (Line 869).

Reply:
I used it in the Signedness parser. Otherwise, I would get a warning since `_pos` is "private".

Reply:
Ah yes, didn't see :+1: